### DISCUSSION TITLE
[FEATURE REQUEST] Add show_changes arg to file.(pre|ap)pend

### DISCUSSION THREAD

**Discussion by boltronics** [saltstack/salt#67533] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
I don't want sensitive lines logged, but would like to use `file.append` or `file.prepend` in my state files.

**Describe the solution you'd like**
I'd like the `show_changes` argument to be available here, similar to `file.managed`.

**Describe alternatives you've considered**
Not using `file.append` or `file.prepend`. This is less than ideal.

**Additional context**
Lots of `file.*` states accept this argument. `file.append` and `file.prepend` should probably accept it too.

**Comment by sagetherage** [comment]
The PR is failing a bunch of tests, but keeping this labeled as `Aluminium` and in the AL milestone to plan for the AL point release

**Comment by meaksh** [comment]
Just recreated the initial PR here: https://github.com/saltstack/salt/pull/69009
